### DISCUSSION TITLE
[Sandbox] Impossible d'utiliser certains moyens de paiement

### DISCUSSION THREAD

**Discussion by pamls** [PayGreen/documentation#10] [Q&A] [answered]
Bonjour,

J'ai tenté de créer une page de paiement avec l'ensemble des moyens de paiement activés sur ma sandbox.
![image](https://github.com/PayGreen/documentation/assets/5923407/cd2e1994-10a6-4898-8e52-09c4e3e9ab58)

mais quand je ne spécifie pas "platform", je n'ai que les cartes et titres restaurant, et lorsque j'ajoute swile, restoflash, ancv, apple_pay, j'obtiens le message d'erreur suivant : `Unauthorized platform : swile, restoflash, ancv, apple_pay`

Est-ce que cela signifie que ces moyens de paiement ne sont pas disponibles en mode sandbox mais qu'il n'y aura aucun problème pour les utiliser en production ?

Merci

**Comment by pielrx** [comment]
Bonjour,

Un incident a été déclaré sur notre environnement de Sandbox qui a certainement causé votre problème.
Pourriez-vous refaire un test ?

**Reply by pamls** [reply]
Bonjour,

Je n'ai pas constaté de changement, je reçois la même erreur quand je mets "swile" par exemple.
```
{
    "type": "https://symfony.com/errors/validation",
    "title": "Validation Failed",
    "detail": "platforms: Unauthorized platform : swile",
    "violations": [
        {
            "propertyPath": "platforms",
            "title": "Unauthorized platform : swile",
            "parameters": []
        }
    ]
}
```

**Reply by pamls** [reply, marked as answer]
@pielrx le problème a été résolu, merci.
